content/tools/json_to_xml.md

```diff
@@ -0,0 +1,171 @@
+---
+ title: "JSON to XML Converter" 
+ description: "JSON to XML Converter."
+ date: 2025-02-25T01:15:25+05:30 
+ draft: false 
+ og_image: "/quickstart/images/django.svg"
+---
+
+<div class="container mt-4">
+  <div class="row mb-3">
+    <div class="col-12">
+      <h1 class="mb-3">JSON to XML Converter</h1>
+      <p class="lead">Convert your JSON into a well-formatted XML. Simply paste your JSON in the left panel and click convert!</p>
+    </div>
+  </div>
+  <div class="row">
+    <!-- JSON Input Card -->
+    <div class="col-lg-6 mb-4 d-flex">
+      <div class="card flex-fill shadow-sm">
+        <div class="card-header">
+          <strong>JSON Input</strong>
+        </div>
+        <div class="card-body d-flex flex-column">
+          <div class="form-group flex-grow-1">
+            <textarea id="jsonInput" class="form-control h-100" style="resize: none;" placeholder='Paste Your Json Here'></textarea>
+          </div>
+          <button id="convertBtn" class="btn btn-primary btn-block mt-2">Convert</button>
+        </div>
+      </div>
+    </div>
+    <!-- XML Output Card -->
+    <div class="col-lg-6 mb-4 d-flex">
+      <div class="card flex-fill shadow-sm">
+        <div class="card-header">
+          <strong>XML Output</strong>
+        </div>
+        <div class="card-body d-flex flex-column">
+          <pre id="output" class="h-100">Your formatted XML will appear here...</pre>
+          <div class="mt-3 d-flex justify-content-between">
+            <button id="copyBtn" class="btn btn-secondary">Copy to Clipboard</button>
+            <button id="downloadBtn" class="btn btn-success">Download XML</button>
+            <button id="clearBtn" class="btn btn-danger">Clear</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div id="alertPlaceholder"></div>
```

Comment:
After the main tool, add more content about the page for SEO. The tool is a 100% free json to xml converter.